go-app/report/details.go

```diff
@@ -159,11 +159,13 @@ func getImportantEvents(cfgFile *Config, fContent string, importantEvents map[in
 				waitGroup.Done()
 				return
 			}
-			ev_content := ev_rgx_comp.FindString(fContent)
-			if ev_content != "" {
-				mutex.Lock() 
-				importantEvents[contentMap[ev_content]] = ev
-				mutex.Unlock() 
+			ev_content := ev_rgx_comp.FindAllString(fContent, -1)
+			if len(ev_content) > 0 {
+				mutex.Lock()
```

Comment:
The mutex should only need to go around the write of the shared resource.

Reply:
Okay. I will solve that